webapp/graphite/urls.py

```diff
@@ -33,8 +33,16 @@
     url('^s/(?P<path>.*)', shorten, name='shorten'),
     url('^S/(?P<link_id>[a-zA-Z0-9]+)/?$', follow, name='follow'),
     url('^$', browser, name='browser'),
+
 ]
 
+try:
+    graphite_urls.append(
+        url('^prometheus/', include('django_prometheus.urls')),
```

Comment:
should it be something like `prometeus-metrics/` ? Thinking about possible (logical) clash with https://github.com/graphite-project/graphite-web/pull/2195

Reply:
`prometheus-metrics/metrics` isn't nice. Maybe having `prometheus/metrics`, `prometheus/write`, `prometheus/read` isn't really an issue ? I could just create a prometheus apps that will later be extended to add `/read` and redirect its `/metrics` to django-prometheus